craft_providers/lxd/lxd_provider.py

```diff
@@ -97,7 +97,7 @@ def launched_environment(
         project_name: str,
         project_path: pathlib.Path,
         base_configuration: Base,
-        build_base: str,
+        build_base: Tuple[str, str],
```

Comment:
As far as I can tell, this is a transparent change for the craft applications, right?

Regardless, the Provider classes are labeled in-developer (and lpcraft isn't using them :see_no_evil:), so we have some wiggle room to make changes.

Can you update the base `Provider` class, the `MultipassProvider` class, and the docstrings?  The linters in the craft applications will catch the mismatched function signatures.

Reply:
This is transparent change unless something try to read the value and expect a str like "22.04".